### ISSUE TITLE
How do you build/iinstall this? 

### ISSUE THREAD

**Issue by hqm** [jlnsrk/h265ify#3]
I know nothing about Microsoft browser or extensions, are there any instructions on how to build or install this extension
given the git repo? thank you

**Comment by ghettopato** [comment]
This was just a fix for when edge supported h265, but didn't announce it to websites. That has been rectified. This extension is useless now.

**Comment by hqm** [comment]
Oh, is h265 supposed to be supported now in Edge? Because I'm having trouble having it play a video that plays fine on Chrome and Safari and Brave. I get a blank video in Edge.  I'm wondering if there's some special extra media type header we need or something.

**Comment by hqm** [comment]
We generate the video from an mkv file of H.265 video, using ffmpeg to generate an mp4.

**Comment by ghettopato** [comment]
> Oh, is h265 supposed to be supported now in Edge? Because I'm having trouble having it play a video that plays fine on Chrome and Safari and Brave. I get a blank video in Edge. I'm wondering if there's some special extra media type header we need or something.

yes sir but you have to install the h265 video extension from the windows store for it to work
